### PR TITLE
Update Jenkinsfile to use edgeXBuildDocker function (fuji)

### PR DESCRIPTION
**Pull request by lranjbar**
Signed-off-by: Lisa Rashidi-Ranjbar <lisa.a.rashidi-ranjbar@intel.com>

### CODE REVIEW

**Review by soda480 — APPROVED** [review]
LGTM

File: Jenkinsfile [hunk 1]
```diff
@@ -14,116 +14,8 @@
...
-            extensions: [[
-                $class: 'SubmoduleOption',
-                recursiveSubmodules: true,
-            ]]]
-        )
-    ) _
-}
\ No newline at end of file
+edgeXBuildDocker (
+    project: 'docker-edgex-consul',
```
Comment (ernestojeda) [inline Jenkinsfile:18]: If both the project and the dockerImageName are the same you can leave out the dockerImageName

**Review by jamesrgregg — APPROVED** [review]
super simple Jenkinsfile

**Review by ernestojeda — CHANGES_REQUESTED** [review]
Just a quick change to the Jenkinsfile

**Review by ernestojeda — APPROVED** [review]

**Pull request merged**